modules/jolt_physics/spaces/jolt_layers.h

```diff
@@ -63,6 +63,7 @@ class JoltLayers final
 
 public:
 	JoltLayers();
+	~JoltLayers();
```

Comment:
I don't know how Godot deals with this generally, but this destructor is technically virtual as well (whether specified or not) so might be good to be explicit about it:

```suggestion
	virtual ~JoltLayers();
```